src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/SentenceMovementService.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.prison.service.enteringandleaving
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderKeyDateAdjustmentRepository
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderSentenceAdjustmentRepository
+
+@Service
+@Transactional
+class SentenceMovementService(
+  private val offenderSentenceAdjustmentRepository: OffenderSentenceAdjustmentRepository,
+  private val offenderKeyDateAdjustmentRepository: OffenderKeyDateAdjustmentRepository,
+) {
+  fun deactivateSentences(bookingId: Long) {
+    deactivateSentenceAdjustments(bookingId)
```

Comment:
interesting, I never knew we did this - it will have an impact in the future when adjustments is migrated off NOMIS